docs/source/notebooks/ModelBuilder in PyMC-Marketing context.ipynb

```diff
@@ -0,0 +1,1149 @@
+{
```

Comment:
<p>No need to justify why we picked the MMM as an example. Just say we pick it, but it works for all pymc MMM and CLV models.</p><p> </p><p>Also you refer to sklearn here and scikit-learn below. Should pick one.</p><p> </p><p>The <code>ModelBuilder</code> can be referenced just at the end of the article, for those interested in the internals, and should include a link to the pymc-experimental docs/ example. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/pymc-marketing/pull/324/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>The notebook was meant to be an introduction of ModelBuilder as a module, so I'm not sure if I agree that it should be just a mention at the end (and this was supposed to be an example for it)</p>

Reply:
<p>The notebook is quite short so I think it's fine, but PyMC-Marketing users don't need to know something called <code>ModelBuilder</code> exists. It's one more piece of cognitive load that doesn't gain them anything. All they care is that PyMC-Marketing models can be configed, saved and loaded imo</p>